ui/plugins/eu.esdihumboldt.hale.ui.common.definition.editors/src/eu/esdihumboldt/hale/ui/common/definition/editors/DefaultPropertyEditor.java

```diff
@@ -257,6 +260,22 @@ public void widgetSelected(SelectionEvent e) {
 			});
 		}
 
+		// add project variable content assistance
+		if (entity != null) {
```

Comment:
I think the condition is not necessary - as stated in the class documentation it may be null if it is unavailable, but should not be needed for the content assist.